rfd/0089-merge-webapps.md

```diff
@@ -0,0 +1,192 @@
+
+#### Actions
+
+- [ ] Triage issues.
+  - [ ] Close those that are no longer relevant.
+  - [ ] [Transfer remaining issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) to the [Teleport repo](https://github.com/gravitational/teleport)
+  - [ ] Add the [ui label](https://github.com/gravitational/teleport/labels/ui) to
+        those transferred issues.
+- [ ] Triage PRs.
+  - [ ] Close those that are no longer relevant.
+  - [ ] Merge remaining PRs.
+  - [ ] Apply the necessary backports.
+- [ ] Update [the default path to tsh in dev mode](https://github.com/gravitational/webapps/blob/27c615b3ff6f317a85fac4aa28b8e73fa4aa0d28/packages/teleterm/src/mainProcess/runtimeSettings.ts#L18-L23) for Connect.
+
+### Webapps.e repository
+
+[repository](https://github.com/gravitational/webapps.e)
+
+#### Actions
+
+- [ ] Triage issues.
+  - [ ] Close those that are no longer relevant.
+  - [ ] [Transfer remaining issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository) to the [Teleport.e repo](https://github.com/gravitational/teleport.e)
+  - [ ] Add the [ui label](https://github.com/gravitational/teleport.e/labels/ui)
+- [ ] Triage PRs.
+  - [ ] Close those that are no longer relevant.
+  - [ ] Merge remaining PR's.
+  - [ ] Apply the necessary backports.
+
+### Teleport repository
+
+[repository](https://github.com/gravitational/teleport)
+
+#### Actions
+
+- [ ] Remove `/webassets` submodule
+  - This submodule is no longer required as the web UI will be built on-demand.
+  - The folder will remain as the output location of the on-demand build but
+    will not be committed.
+- [ ] Clone the [Webapps repository](https://github.com/gravitational/webapps) into
+      the Teleport root. [Maintaining their respective git histories](https://stackoverflow.com/questions/13040958/merge-two-git-repositories-without-breaking-file-history)
+  - [ ] This will need to be done for every respective version branch (v9, v10, v11)
+- [ ] Update targets that use the `packages/webapps.e` submodule to point points to
+      the correct version in the `teleport.e/web` folder.
+- [ ] Only require teleport build processes to run on teleport paths and the webapp
+      ones to run on the webapp paths
+- [ ] Archive the [Webapps repository](https://github.com/gravitational/webapps).
+- [ ] Update Connect's build pipelines (in teleport/dronegen) as webapps will no longer need to be cloned.
+
+#### CI jobs
+
+- **CodeQL / Analyze (go) (pull_request)**
+- **Teleport-IntegrationTest (ci-account)**
+- **Teleport-UnitTest (ci-account)**
+- **Teleport-DocTest (ci-account)**
+- **Teleport-Lint (ci-account)**
+- **Teleport-OsCompatibility (ci-account)**
+  - Disable for changes exclusively made in the `/web` path.
+- **Assign / Auto Request Review**
+  - Create a list of full stack engineers and have it pick from this list for
+    changes made to the `/web` path.
```

Comment:
How do we feel about sticking to the current process of manually picking reviewers?

It's not something we've discussed within the frontend team and as I mentioned [in my earlier comment](https://github.com/gravitational/teleport/pull/16413#discussion_r975315309), I feel it might be disruptive for the day-to-day work.